build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '0.0.85'
+version '0.0.86-CANDIDATE'
```

Comment:
Could we change the version number so it doesn't contain Candidate?

Reply:
done